lib/ht_sip_validator/validation/image/count.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module HathiTrust::Validation::Image
+  class Count < HathiTrust::Validation::Base
+
+    def perform_validation
+      image_files = HathiTrust::Validation::Image.image_files(@sip.files)
+      
+      if image_files.count != text_files.count
```

Comment:
In that case, do we want to:
- Error on images count is less than text count.  
- Warn when images count is greater than text count.